packages/helpers/qrcode-modal/src/browser/languages/es.ts

```diff
@@ -1,8 +1,8 @@
 export default {
-  choose_preferred_wallet: "Elige tu cartera preferida",
-  connect_mobile_wallet: "Conectar a cartera móvil",
-  scan_qrcode_with_wallet: "Escanee el código QR con una cartera compatible con WalletConnect",
-  return_to_mobile_options: "Regresar a las opciones de cartera móvil",
+  choose_preferred_wallet: "Elige tu billetra preferida",
+  connect_mobile_wallet: "Conectar a billetra móvil",
+  scan_qrcode_with_wallet: "Escanea el código QR con una billetra compatible con WalletConnect",
+  return_to_mobile_options: "Regresar a las opciones de billetra móvil",
   view_qrcode_option: "Ver código QR en vez",
```

Comment:
hey that's great! but I think this line could be changed: view_qrcode_option: "Ver código QR"

Reply:
Thanks 🙌 I will update it

Reply:
I agree with @madkingfisher. Everything looks good except for that "en vez".

Reply:
Changed 👍